docs/sources/alerting/_index.md

```diff
@@ -37,8 +37,10 @@ The actual notifications are configured and shared between multiple alerts.
 ## Alert execution
 
 Alert rules are evaluated in the Grafana backend in a scheduler and query execution engine that is part
-of core Grafana. Only some data sources are supported right now. They include `Graphite`, `Prometheus`, `Loki`, `InfluxDB`, `Elasticsearch`,
-`Google Cloud Monitoring`, `Cloudwatch`, `Azure Monitor`, `MySQL`, `PostgreSQL`, `MSSQL`, `OpenTSDB`, `Oracle`, and `Azure Data Explorer`.
+of core Grafana. Alert rules can query only backend data sources with alerting enabled. Such data sources are:
+- the builtin ones: `Graphite`, `Prometheus`, `Loki`, `InfluxDB`, `Elasticsearch`,
+`Google Cloud Monitoring`, `Cloudwatch`, `Azure Monitor`, `MySQL`, `PostgreSQL`, `MSSQL`, `OpenTSDB`, `Oracle`, and `Azure Data Explorer`
```

Comment:
Unless this changed recently, `Azure Data Explorer` is a plugin, but one we own.

There are also our enterprise plugins that support alerting, I'm not sure if that should go here or not.